fossa/info/action.yml

```diff
@@ -67,3 +79,38 @@ runs:
         echo "head-revision=${HEAD_REVISION}"
       } >> $GITHUB_OUTPUT
     shell: bash
+  - name: Get additional PR info
+    if: ${{ steps.info.outputs.is-pull-request == 'true' }}
+    id: info-pr
+    env:
+      BASE_REF: ${{ steps.info.outputs.base-ref }}
+      BASE_REVISION: ${{ steps.info.outputs.base-revision }}
+      GH_TOKEN: ${{ inputs.github-token }}
+    run: |
+      # Check recent commits of the base branch for existing scanning results
+      # history depth to search for scanning results
+      depth=5
+      # Ensure history is available for the base branch up to the specified depth
+      git fetch origin "${BASE_REF}" --depth="$depth"
+      # Check if the base branch or its recent parents have scanning results
+      base_ref_with_scanning_results=""
+      for i in $(seq 0 $((depth-1))); do
+        commit=$(git rev-parse "${BASE_REVISION}~$i" 2>/dev/null)
+        # Check if a workflow run of "Check Licenses" succeeded for the commit
+        if gh run list --commit "$commit" --json workflowName,conclusion \
+          --jq '[.[] | select(.workflowName == "Check Licenses" and .conclusion == "success")] | any' | grep -q true; then
```

Comment:
can we guarantee that the workflow name for the fossa check is always equal to "Check Licenses"?
If we want to rely on that we should definitely document that in the calling workflows with some kind of warning, so teams are aware of that.

Maybe also overkill and wildly inefficient, but in a combination with `gh run view <run-id> --log` and some grep magic we could get info if the action has actually been called?

Reply:
Or maybe even easier: The context should have the workflow name available as `${{ github.workflow }}`

Reply:
Good catch! `${{ github.workflow }}` should do the trick